templates/Includes/SearchResult.ss

```diff
@@ -0,0 +1,8 @@
+<article class="listing__item" data-highlight="$Up.Query.ATT">
```

Comment:
just a reminder to remove the $Up from this line

Reply:
This template has been removed